### DISCUSSION TITLE
Early Modern English

### DISCUSSION THREAD

**Discussion by clumperton** [psb1558/Junicode-font#269] [Q&A] [answered]
I use XeLaTeX and polyglossia, and in a document I have Junicode set up as the font for various languages. I see a wonderful example in the Junicode language samples file of "early modern English" and I would like to use that. I have read through the polyglossia and junicode manuals, but I'm still lost. I've tried:
\setotherlanguage{earlymodernenglish}

\newfontfamily\earlymodernenglishfont{Junicode}

but XeLaTeX simply prints regular English.

**Comment by kenmcd** [comment, marked as answer]
That text is normal modern English - with Stylistic Set 8 applied (`ss08`).
See: **4.3.9. `ss08` Contextual Long s** in the manual (page 30 in v2.206 manual).
In the `language_samples.tex` file go down to line 759 to see it applied.
